CHANGES.rst

```diff
@@ -13,6 +13,22 @@ Changelog
 
 .. towncrier release notes start
 
+3.11.0 (2023-11-08)
+===================
+
+
+Features
+--------
+
+- Added pulpcore 3.40 compatibility.
+  `#40 <https://github.com/pulp/pulp_python/issues/40>`__
```

Comment:
This probably needs fixing by hand on the 3.11 branch.

Reply:
Here we go: https://github.com/pulp/pulp_python/pull/632